platforms/rpi/src/main.cpp

```diff
@@ -83,7 +86,13 @@ void setup(int argc, char **argv) {
     double lat = 0.0f;
     double lon = 0.0f;
     std::string scene = "scene.yaml";
-    const std::string& apiKey = "vector-tiles-tyHL4AY";
+
+    const char* apiKey = std::getenv("MAPZEN_API_KEY");
```

Comment:
Since we already have access to the API key in CMake, we can just pass it in at compile-time as a `#define` rather than checking an environment variable at run-time. That is, in CMake we say `add_definitions(-DMAPZEN_API_KEY="$MAPZEN_API_KEY")` and in this file we just use the value of `MAPZEN_API_KEY` (I may not have the syntax exactly right).